### PR TITLE
Lesson 4 - Kubernetes

### PR DESCRIPTION
**Pull request by thomasaelbrecht**
Copied mostly from Chamilo from last year. All video's on top are new!

I have some questions:

- I have some doubts about the minikube videos quality, although they are from Google. What do you think?
- Should we add more videos about the different controllers (Deployments, ReplicaSets, Jobs...)?

Part of #13.

### CODE REVIEW

File: docs/04-kubernetes.md [hunk 1]
```diff
@@ -1 +1,41 @@
 # Kubernetes
+
```
Comment (MartenBE) [inline docs/04-kubernetes.md:2]: I would add a requirements section that lists what we expect them to understand and be able to do.

**Review by MartenBE — CHANGES_REQUESTED** [review]

File: docs/04-kubernetes.md [hunk 2]
```diff
@@ -1,41 +1,72 @@
...
+    -   CronJobs
+-   You should be able to organise applications using labels and selectors.
+-   You can install `minikube`, start a local Kubernetes cluster, and use the `kubectl` command to iteract with it.
+-   You can explain the contents of a given Kubernetes manifest file, and you can apply it to a Kubernetes cluster.
+
 ## Theory
 
+### Articles
+
+TODO:
```
Comment (MartenBE) [inline docs/04-kubernetes.md]: If there are no articles, you can omit this subsection.

**Review by MartenBE — CHANGES_REQUESTED** [review]

File: docs/04-kubernetes.md (hunk 2 above)
Comment (thomasaelbrecht) [reply docs/04-kubernetes.md]: I had one article, but hadn't got the time to put it here.

**Comment by thomasaelbrecht** [comment]
Now everything should be fine. Now you can review it again 😉

**Review by MartenBE — APPROVED** [review]
I've removed the TODO articles subsection, otherwise looks good.

**Pull request merged**